MollieShopware.php

```diff
@@ -547,4 +552,11 @@ private function addIndexToTransactions()
         }
     }
 
+    private function cleanOrdermailVariables()
+    {
+        /** @var Connection $connection */
+        $connection = $this->container->get('dbal_connection');
+
+        $connection->executeQuery('UPDATE mol_sw_transactions set ordermail_variables = NULL');
```

Comment:
just a little something... i'd prefer to see the SET keyword in uppercase as well as the UPDATE already is